packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -140,6 +146,17 @@ class OverlayEntry implements Listenable {
   /// The currently mounted `_OverlayEntryWidgetState` built using this [OverlayEntry].
   ValueNotifier<_OverlayEntryWidgetState?>? _overlayEntryStateNotifier = ValueNotifier<_OverlayEntryWidgetState?>(null);
 
+  /// Dispatches event of object creation to [MemoryAllocations.instance].
+  void _maybeDispatchObjectCreation() {
```

Comment:
we should probably think about how not to duplicate this method

Reply:
Yes, there are plans and discussions around this: http://flutter.dev/go/introduce-disposable

While we do not have agreement, we are duplicating code. It will be cleaned up. My task is to research different options.
For now I do not want to block leak cleanup, that is done by external developers, in Flutter Framework.

Reply:
should we add a todo then?